content/blog/tidymodels-2021-q1/index.Rmd

```diff
@@ -0,0 +1,141 @@
+---
+output: hugodown::hugo_document
+
+slug: tidymodels-2021-q1
+title: Catch up with tidymodels
+date: 2021-03-10
+author: Julia Silge
+description: >
+    Recent tidymodels releases offer new functions for easier model building and
```

Comment:
Maybe make the timeline explicit here — i.e. This update covers the most important changes in Q1 of 2021.